PokeAlarm/Filters/BaseFilter.py

```diff
@@ -191,3 +227,30 @@ def __call__(self, filtr, event):
             filtr.reject(event, self._attr_name, value, self._limit)
 
         return result
+
+
+class CheckTime(object):
+    """ Function used to check if a timestamp passes or not. """
+
+    def __init__(self, min_time, max_time):
+        self._min_time = min_time
+        self._max_time = max_time
+
+    def __call__(self, filtr, event):
+        now = datetime.now()
+        current_time = (now - now.replace(hour=0, minute=0,
+                        second=0)).total_seconds()
+        if self._min_time <= self._max_time:
```

Comment:
It feels like having a state of `min_time == max_time` here would cause a failure in the logic. You should probably split these out into separate evaluations. What is the expected outcome if someone sets both filters to be the same?

Reply:
Logically, if you set any time to X:XX, we assumes that means X:XX:00.
Now, if you set a time range with the same minimum and maximum (which means a range from X:XX:00 to X:XX:00), the only alarm which passes has to be on the exact same second (so X:XX:00). For any other time, the filter will reject the event. So I would say the expected outcome is logic.

What do you think about this behaviour?

Reply:
That makes sense.